version.json

```diff
@@ -2,7 +2,7 @@
     "version": [
         24,
         0,
-        1
+        2
     ],
-    "version_string": "24.0.1"
+    "version_string": "24.0.2"
```

Comment:
We're removing a bunch of public functions, so this should probably be `25.0.0`.

Reply:
rgr